keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1480,16 +1461,17 @@ If we were to offer a way to lookup arbitrary other resources, or even if
 we provided selective access to just some resources, this might become
 easier. This can explored as future work.
 
-#### Access to namespace metadata
+#### Access to namespace
 
-We have general agreement to include this as a feature, but need to provide
-a concrete design.
+We have general agreement to grand CEL expression access to namespace through a newly added CEL variable `namespaceObject`.
+If the resource is cluster scoped, `namespaceObject` will be null.
 
-- Namespace labels and annotations are the most commonly needed fields not
-  already available in the resource being validated. Note that
-  namespaceSelectors already allow matches to examine namespace levels, but we
-  also have use cases that need to be able to inspects the fields in CEL
-  expressions.
+`namespaceObject` will provide access to all existing fields under namespace metadata, namespace spec and namespace status except for metadata.managedFields and metadata.ownerReferences.
```

Comment:
if we use the same constructs later for mutation, the namespace finalizers seem likely targets for both read and write.

Reply:
Yes and we will expose finalizers from both namespace.metadata.finalizers and namespace.spec.finalizers

Reply:
Conceptually, is this the same as `namespaceObject` will provide access to the same fields that are accessible to a CEL expression admissing the namespace object directly?  For consistency, I think this might be what we want.

Reply:
> Conceptually, is this the same as namespaceObject will provide access to the same fields that are accessible to a CEL expression admissing the namespace object directly? For consistency, I think this might be what we want.

I like that.  Downsides?